tools/raisinbread_refresh.php

```diff
@@ -0,0 +1,286 @@
+array_walk($rbInstrumentTables, 'runPatch');
+
+// Import Raisinbread data
+echo <<<INFO
+[*] Importing Raisinbread data....
+
+INFO;
+$rbData= glob("../raisinbread/RB_files/*.sql");
+array_walk($rbData, 'runPatch');
+
+// Restore config settings if they were successfully found before.
+if (isset($urlConfigSetting)) {
+    restoreConfigSetting('url', $urlConfigSetting);
+}
+if (isset($baseConfigSetting)) {
+    restoreConfigSetting('base', $baseConfigSetting);
+}
+if (isset($hostConfigSetting)) {
+    restoreConfigSetting('host', $hostConfigSetting);
+}
+
+// Trigger a password reset because the password for `admin` in the Raisinbread
+// database is public.
+echo 'Please choose a new password for the admin user:' . PHP_EOL;
+runCommand('php resetpassword.php admin');
+
+
+// END SCRIPT
+
+
+/**
+ * Drops core LORIS tables as well as Raisinbread instrument tables
+ */
+function dropTables(): void
+{
+    runPatch(
+        '../raisinbread/instruments/instrument_sql/9999-99-99-drop_instrument_tables.sql'
+        );
+    runPatch('../SQL/9999-99-99-drop_tables.sql');
+}
+
+function runPatch(string $file): void
+{
+    global $mysqlCommand;
+    runCommand(
+        "cat $file | $mysqlCommand"
+    );
+}
+
+/**
+ * A wrapper around `exec()` built-in function with basic error reporting.
+ *
+ * @param string $command Bash command to be executed by `exec()`
+ * jj
+ *
+ * @return void. Causes script to exit on non-successful status code.
+ */
+function runCommand(string $command): void
+{
+    global $password;
+    // Hide password from output
+    $output = str_replace($password, str_repeat('*', 10), $command);
+    echo "Running command: `$output`" . PHP_EOL;
+    exec($command, $output, $status);
+    // If a non-zero exit code is given, then an error has occurred.
+    // In this case, print the output.
+    if ($status) {
+        foreach ($output as $line) {
+            echo $line . PHP_EOL;
+        }
+    }
+}
+
+/**
+ * Update a config setting in LORIS to $value.
+ *
+ * @param string $name Name of config setting in ConfigSettings table.
+ * @param string $value Value to be changed in the Config table.
+ *
+ * @return void
+ *
+ * @throws \DatabaseException
+ */
+function restoreConfigSetting(string $name, string $value): void 
+{
+    echo "Restoring config setting `$name` to value `$value`"
+        . PHP_EOL . PHP_EOL;
+    try {
+        global $DB;
+        $DB->run(
+            "UPDATE Config c 
+            SET c.Value='$value'
+            WHERE c.ConfigID
+            IN (SELECT cs.ID FROM ConfigSettings cs WHERE cs.Name = '$name')"
+        );
+    } catch (\DatabaseException $e) {
+        echo "Couldn't config setting. " .
+            "This may need to be manually if your LORIS is not hosted at " .
+            "localhost." .
+            PHP_EOL;
+    }
+}
+
+function dropRemainingTables(array $tables) {
+    global $mysqlCommand;
+    $commands = ["SET FOREIGN_KEY_CHECKS = 0;"];
```

Comment:
is this a new way to add to an array? Same for L270

Reply:
This initializes an array with one element. Line 270 had a typo -- thanks for drawing attention to it. I've fixed it now.